bindings/python/examples/client/mqtt.py

```diff
@@ -0,0 +1,29 @@
+from iota_sdk import Client, init_logger
+from dotenv import load_dotenv
+import os
+import json
+
+load_dotenv()
+
+node_url = os.environ.get('NODE_URL', 'https://api.testnet.shimmer.network')
+
+# Create a Client instance
+client = Client(nodes=[node_url])
+
+received_events = 0
+
+
+def callback(event):
+    event_dict = json.loads(event)
+    print(event_dict)
+    global received_events
+    received_events += 1
+
+
+# Topics can be found here https://studio.asyncapi.com/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0028/event-api.yml
+client.listen(["blocks"], callback)
+
+# Exit after 10 received events
+while True:
```

Comment:
Is....is this for real the best way?

Reply:
Probably not the best way, but it's a simple one, don't know what's better for an example

Reply:
Well we certainly don't want to have an example with a really bad method in it. Is the callback executed concurrently? Is there a sync primitive we can use to do this properly?

Reply:
But this loops looks like a 100% CPU until we have 10 events

Reply:
I used `threading.Event()` now which seems to be an efficient way of doing it